src/lib.rs

```diff
@@ -132,6 +132,7 @@ extern crate rand;
 extern crate x86intrin;
 extern crate parquet_format;
 extern crate chrono;
+extern crate lazy_static;
```

Comment:
Do we need to add lazy_static to lib.rs even if it is for benchmarks only?

Reply:
Good point. I think both lazy_static and rand are only used in tests & benchmarks. Let me move them to dev-dependencies.